integration/spark/shared/src/main/java/io/openlineage/spark/agent/facets/builder/SparkPropertyFacetBuilder.java

```diff
@@ -36,14 +52,31 @@ public SparkPropertyFacetBuilder(OpenLineageContext context) {
 
   @Override
   protected void build(
-      SparkListenerJobStart event, BiConsumer<String, ? super SparkPropertyFacet> consumer) {
+      SparkListenerEvent event, BiConsumer<String, ? super SparkPropertyFacet> consumer) {
+    consumer.accept("spark_properties", buildFacet(event));
+  }
+
+  public SparkPropertyFacet buildFacet(SparkListenerEvent event) {
     Map<String, Object> m = new HashMap<>();
```

Comment:
what is `m`?

Reply:
I didn't change code here. It is named by the previous authors. I think 'm' means properties in SparkPropertyFacet. If this name is not proper, I can change it in my PR.

Reply:
I didn't change code here. It is named by the previous authors. I think 'm' means properties in SparkPropertyFacet. If this name is not proper, I can change it in my PR.